com.unity.render-pipelines.universal/Runtime/2D/Renderer2D.cs

```diff
@@ -29,6 +29,9 @@ internal class Renderer2D : ScriptableRenderer
 
         Renderer2DData m_Renderer2DData;
 
+        PostProcessData m_RendererPostProcessData;
+        PostProcessData m_CurrentPostProcessData;
```

Comment:
Is this shared between renderers? Maybe could be moved to ScriptableRenderer

Reply:
K